index.js

```diff
@@ -30,13 +30,26 @@ function injectAsync(source) {
             switch (opts.type) {
                 case undefined:
                 case 'tile':
-                    source.getTile(opts.z, x, y, (err, data, hdrs) => {
+                    var cb = function(err, data, hdrs) {
                         if (err) {
                             reject(err);
                         } else {
                             resolve({data: data, headers: hdrs});
                         }
-                    });
+                    };
+
+                    // fallback for tilelive-vector
+                    cb.format = opts.format;
```

Comment:
P.S.: I'm not sure if the parameter forwarding should be more verbose by adding every property of `opts` as property to the callback..... I just took those who were a part of the original hack on the kartotherian fork or were added later to tilelive-vector.